client.go

```diff
@@ -23,7 +23,7 @@ import (
 	"sync"
 
 	"github.com/pkg/errors"
-	"github.com/IhorBondartsov/go/soaphlp"
```

Comment:
Please don't change the import path for PRs.

The easiest is to check out at the original path ($GOPATH/src/github.com/tgulacsi/go) and add another remote (git remote add fork git@github.com:IhorBondartsov/go), then `git push fork master`

Reply:
Thank you, I understood. I will try to change this problem.

Reply:
I sent new PRs, so i think you can close this PRs.